contrib/ansible/roles/skydive_common/tasks/config.yml

```diff
@@ -26,6 +26,69 @@
     key: analyzers
     value: "{{ analyzers.split(',') }}"
 
+- yedit:
+    src: "{{ skydive_config_file }}"
+    key: "auth.type"
+    value: "{{ skydive_auth_type }}"
+    state: present
+  when: skydive_auth_type is defined
+
+- yedit:
+    src: "{{ skydive_config_file }}"
+    edits:
+        - key: "auth.keystone.auth_url"
+          value: "{{ os_auth_url }}"
+        - key: "auth.keystone.tenant_name"
+          value: "{{ os_tenant_name }}"
+        - key: "auth.analyzer_username"
+          value: "{{ os_username }}"
+        - key: "auth.analyzer_password"
+          value: "{{ os_password }}"
+        - key: "auth.keyston.tenant_name"
```

Comment:
should be removed   key: "auth.keyston.tenant_name" as it's misspell and you already set auth.keystone.tenant_name